### PR TITLE
STABmons: Fix move validation error

### PR DESCRIPTION
**Pull request by KrisOnline**
http://replay.pokemonshowdown.com/gen7stabmons-654178506

Pokemon were allowed to use banned moves still.

### CODE REVIEW

File: sim/team-validator.js [hunk 1]
```diff
@@ -901,7 +901,8 @@ class Validator {
 			alreadyChecked[template.speciesid] = true;
 			if (dex.gen === 2 && template.gen === 1) tradebackEligible = true;
 			// STABmons hack to avoid copying all of validateSet to formats
-			if (ruleTable.has('ignorestabmoves') && format.noLearn.indexOf(moveid) < 0 && !move.isZ) {
+			let noLearn = format.noLearn || dex.getFormat('gen7stabmons').noLearn;
+			if (ruleTable.has('ignorestabmoves') && noLearn.indexOf(move.name) < 0 && !move.isZ) {
```
Comment (Zarel) [inline sim/team-validator.js]: > `noLearn.indexOf(move.name) < 0`

...standard is to use `!noLearn.includes(move.name)`

Comment (KrisOnline) [reply sim/team-validator.js]: Ah my bad, this is just how Sketchmons did it so I copied, will make a PR to change it

Comment (Zarel) [reply sim/team-validator.js]: Change Sketchmons too, if it still exists.

Comment (KrisOnline) [reply sim/team-validator.js]: It doesn’t, stabmons replaced it

**Pull request merged**